bool gpsSession
bool sim
bool fixed